pomchecker-cli/src/main/java/org/kordamp/maven/checker/cli/internal/PomParser.java

```diff
@@ -85,8 +88,21 @@ private static MavenProject createMavenProject(File pomFile, Context context, Pl
                 mavenExecutionRequest.getProjectBuildingRequest();
 
             projectBuildingRequest.setRepositorySession(context.repositorySystemSession());
-            projectBuildingRequest.setRemoteRepositories(context.remoteRepositories()
-                    .stream().map(PomParser::toArtifactRepository).collect(Collectors.toList()));
+            List<ArtifactRepository> remoteRepositories = context.remoteRepositories()
+                    .stream().map(PomParser::toArtifactRepository).collect(Collectors.toList());
+
+            // Add deployment repository to resolve the artifacts in the project
+            MavenArtifactRepository thisRepository = new MavenArtifactRepository();
+            thisRepository.setId("deployment");
```

Comment:
I think it would be better to prefix this with `pomchecker_` to avoid clashes with user defined repositories found in `settings.xml`